ecm-wcm-extension/src/main/webapp/WEB-INF/conf/social/integration-configuration.xml

```diff
@@ -399,73 +411,6 @@
       </init-params>
     </component-plugin>
   </external-component-plugins>
-  <external-component-plugins>
```

Comment:
Just a question, you removed those plugins because they are duplicated ?

Reply:
yes, and that may caused a problem, thus he plugins can't locate the groovy templates